resources/views/content/home.blade.php

```diff
@@ -36,10 +36,10 @@
         </section>
     </context>
 
-    <section class="container mx-auto flex flex-wrap">
-    @foreach($packagist->except(['astrotomic/laravel-translatable'])->sortByDesc('downloads.total') as $package)
-        <package :package="$package" :github="$github" />
-    @endforeach
+    <section class="container mx-auto grid grid-cols-2">
```

Comment:
On smaller devices we only have one column of packages.
```suggestion
    <section class="container mx-auto grid grid-cols-1 lg:grid-cols-2">
```